interventions/BroadcastEventToOtherNodes.cpp

```diff
@@ -115,8 +115,8 @@ namespace Kernel
             }
             else
             {
-                const std::vector<suids::suid>& r_nodes = p_node_context->GetMigrationInfo()->GetReachableNodes();
-                const std::vector<MigrationType::Enum>& r_mig_types = p_node_context->GetMigrationInfo()->GetMigrationTypes();
+                const std::vector<suids::suid>& r_nodes = p_node_context->GetMigrationInfo()->GetReachableNodes( Gender::MALE );
+                const std::vector<MigrationType::Enum>& r_mig_types = p_node_context->GetMigrationInfo()->GetMigrationTypes( Gender::MALE );
```

Comment:
probably need a comment on why male...hmm.  or change these methods to have a default value of MALE?